src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml

```diff
@@ -253,6 +256,12 @@
                     <Insets top="4.0"/>
                   </VBox.margin>
                 </CheckBox>
+                <CheckBox mnemonicParsing="false" text="Only One Winner per Round"
```

Comment:
Is "Allow only one winner per round" a better description? "Force only one winner per round"? If so, I'd refactor all `*OnlyOneWinnerPerRound` variables accordingly. Or maybe I'm off base and this is fine.

Reply:
Fine by me either way.  I'll throw out there the concise "One winner per round"

Reply:
I think "allow only one winner per round" is good.